test/test_moduleLoad.py

```diff
@@ -10,12 +10,12 @@
 class Test_load_connector_map(TestCase):
     def test__load_connector_map(self):
         # noinspection PyUnresolvedReferences
-        import sswsdk
+        import swsssdk
 
     def test__db_map_attributes(self):
-        import sswsdk
-        db1 = sswsdk.SonicV1Connector()
+        import swsssdk
+        db1 = swsssdk.SonicV1Connector()
```

Comment:
OK. I created an issue for it: https://github.com/Azure/sonic-py-swsssdk/issues/5